DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionInterceptor.swift

```diff
@@ -98,7 +98,7 @@ public struct URLSessionInterceptor {
     // MARK: - Private
 
     private func extractTraceID(from request: URLRequest) -> TraceID? {
-        guard let headers = request.allHTTPHeaderFields else {
+        guard let headers = request.allHTTPHeaderFields else { // swiftlint:disable:this unsafe_all_http_header_fields
```

Comment:
💡 It is IMO fine to ignore this warning here as `URLSessionInterceptor` is only used by Alamofire extension, which should be deprecated and removed. We have no crash reports pointing this usage, hence this should be safe.

Alternatively, we could use `value(forHTTPHeaderField:)` to read (one by one) all headers required by DD, B3 and W3C header readers in following lines.

Reply:
I prefer the alternative mentioned, it's clearer 👍 thanks for the change :)